### ISSUE TITLE
Way to stop interception of console.log?

### ISSUE THREAD

**Issue by rayterrill** [tjanczuk/iisnode#583]
Running v0.2.21, setting loggingEnabled="false" in my web.config. I'm no longer getting log files in my log directory, but I'm also not seeing the console.log entries in the browser.

Is there a way to stop/disable the interception of console.log entirely?

**Comment by dupuis2387** [comment]
why would you see it in the browser? node.js is running server side code, not client side code. so you wouldn't see anything in your browser's console....

**Comment by rayterrill** [comment]
You're right, sorry for the noob-ness of the question. In the process of trying to learn Node and Angular, and clearly still mixed up with the server vs client sides of things. :(
